actionpack/lib/action_dispatch/middleware/exception_wrapper.rb

```diff
@@ -10,6 +10,7 @@
 module ActionDispatch
   class ExceptionWrapper
     cattr_accessor :rescue_responses, default: Hash.new(:internal_server_error).merge!(
+      "ActionView::Template::Error"                        => :internal_server_error,
```

Comment:
🤔 Shouldn't this be the default? From the previous line's `Hash.new(:internal_server_error)` I would expect `rescue_responses["ActionView::Template::Error"]` to already return `:internal_server_error`

Reply:
I can see this is confusing, but the `rescue_response?` method requires the key to be present:
https://github.com/rails/rails/blob/d388ed5d36a7e2b3491ccc102b4ab91c6b926d2b/actionpack/lib/action_dispatch/middleware/exception_wrapper.rb#L195-L197

I need to check if this fix is still the prefered behaviour though.